client/src/css/overview.css

```diff
@@ -3,4 +3,92 @@
   height: 80px;
   width: 80px;
   border-radius: 50%;
-}
\ No newline at end of file
+}
+
+.style-thumbnail:hover {
```

Comment:
I see there are some styles that can be shared across other widgets. It might be good to spend ~30 minutes to review each other's widgets and pick these out. 

i.e. the style-thumbnail hover just contains a hover, which can be used in other areas too (unless you plan to add more to this)

Reply:
Good call. I wasn't sure of the best way to make this more generic. In general, there will be styling common to clickable elements, e.g. like here, we may want to cursor to become a pointer on hover for lots of things. One way I could think of to do this was to add a 'clickable' class to anything clickable, and then style the .clickable class. This makes the CSS much cleaner, but it does mean that we'll have to remember to add the 'clickable' class in HTML/JSX to any clickable element. That seemed to be shifting the logic instead of simplifying it. I'm open to either, and I think there's something to be said for having clean, intelligible CSS wherever possible.